api_docs/markdowns/api/bql/bql-aggs-query.md

```diff
@@ -20,18 +20,28 @@ A `BQLAggsQuery` is composed of a list of `BQLAggregate` and an optional `BQLFil
 
 ### Group-Bys
 A group-by is defined by:
-  - a `field` on which the group-by is performed.
-  - some optional `ranges` that define buckets for the group-by operation.
+  - distinct fields on which the group-by is performed.
+  - ranges that define buckets for the group-by operation.
 
-#### Simple GroupBy
-Only [[aggregables fields;analysis-datamodel?filter=agg:]] can be used for group by operations.
+#### Distinct GroupBy
+Only [[aggregables fields;analysis-datamodel?filter=agg:]] can be used for distinct group-by operations. They are specified either by field name or by field, result size and sort order. By default, at most 100 results are returned sorted by value.
+The order key can be `value` or `count`.
 
-**Example**
+**Examples**
 The following groups URLs by their `http_code`.
 ```JSON
 "http_code"
 ```
```

Comment:
Maybe just add a small part that says that: 

```
  "http_code"
```

is actually transformed to: 

```
"distinct": {
  "field": "http_code",
  "size": 10,
  "order": {"value": "asc"}
}
```

by the backend ?

Just to be clear that the string-only version of this call is basically an alias for a default 'distinct' aggregation.